modules/ribbit/http.py

```diff
@@ -40,9 +41,19 @@ async def static(request):
             headers["Cache-Control"] = "public, max-age=604800, immutable"
 
         return data, 200, headers
-
-    @app.get("/api/config")
-    async def config_list(req):
+    
+    @app.route("/api/sensors")
+    @with_websocket
+    async def sensor_status(request, ws):
+        while True:
+            ret = collections.OrderedDict()
+            for sensor in registry.sensors.values():
+                ret[sensor.config.name] = sensor.export()
+
+            await ws.send(json.dumps(ret))
```

Comment:
I think polling on the client side makes more sense here than websockets.

But, if we want websockets:

* This loop probably wants a `await asyncio.sleep_ms(xxxx)` somewhere?
* It would be great to just store the last measured value somewhere instead of polling the sensors directly

Reply:
Totally, I think I overcomplicated things by polling the device directly as well. Now planning to just get the desired values directly from the sensor (they're already stored, for dps310 it's self.temperature and self.pressure, etc.).

Reply:
What do you mean by "polling on the client side"?

Reply:
I mean just doing an API request regularly (via `setInterval()`).

Reply:
Not only it is simpler, it also avoid having to store state on the device side, which is quite limited in memory.

Reply:
Because I feel like I have limited bandwidth, perhaps we can use websockets for now and iterate on this point if we so desire